app/home/views/home-login.tpl.jade

```diff
@@ -18,4 +18,10 @@ div#form-container(layout="row" layout-align="center center" flex)
             input(type="password" name="password" ng-model="homeLogin.credentials.password" required)
             div.validationMessage(ng-messages="loginForm.password.$error" ng-show="loginForm.password.$touched || loginForm.$submitted")
               div(ng-message="required") Password is required
-          md-button.md-raised.md-primary(type="submit" ng-click="loginForm.$valid ? homeLogin.login() : null") Login
\ No newline at end of file
+          md-button.md-raised.md-primary(type="submit" ng-click="loginForm.$valid ? homeLogin.login() : null") Login
+          div - Or -
+          md-button.md-raised(ng-click="homeLogin.githubLogin()")
+            div(layout="row" layout-align="center")
```

Comment:
I think you can rearrange the content of the row layout in three div flex for consistency.